frontend-react/src/hooks/network/History/ReportsHooks.ts

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from "react";
+
+import { useSessionContext } from "../../../contexts/SessionContext";
+import {
+    BasicAPIResponse,
+    createRequestConfig,
+} from "../../../network/api/NewApi";
+import ReportsApi, { TempRSReport } from "../../../network/api/History/Reports";
+import useRequestConfig from "../UseRequestConfig";
+
+const useReportsList = () => {
+    const { memberships, oktaToken } = useSessionContext();
+    const adminSafeOrgName = useMemo(
+        () =>
+            memberships.state.active?.parsedName === "PrimeAdmins"
+                ? "ignore"
```

Comment:
Would this replace or enhance current Admin org behavior? Like...what if I do want to query just for `md-phd` rather than `*`?

Reply:
Yeah, it would enhance some APIs so they can be used specifically for PrimeAdmins to span multiple orgs with a single call OR they can be used by Org Admins to just see their own data.

Reply:
The dual use throws a yellow flag in the brain at first glance; possibly leading to inconsistencies or tricky code? Just thinking about "if prime admin, if org admin", and "if both use prime admin" being some of the necessary conditions. If you try it out tag me!